data/pizza.json

```diff
@@ -1257,7 +1245,7 @@
 			"ingredients": [
 				"fetaost", "oliver", "lök", "majs", "färska tomater", "gurka", "isbergssallad", "valfri sås"
```

Comment:
[Fårost and vitlökssås](https://sannegarden.com/meny/johanneberg/#:~:text=89Kr-,L%C3%B6k%2C%20Oliver%2C%20Vitl%C3%B6kss%C3%A5s%2C%20F%C3%A5rost%2C%20Gurka%2C%20Isbergssallad%2C%20Majs%2C%20Tomater,-Tonfisk%20sallad)